CONTRIBUTING.md

```diff
@@ -77,6 +77,17 @@ make test
 env EXTRA_CARGO_ARGS=$TESTNAME make test
 ```
 
+Alternatively, you can use [nextest](https://github.com/nextest-rs/nextest) to run tests by setting these environment variables beforehand:
+
+```bash
+# Change the default test runner.
+export CARGO_TEST_COMMAND="nextest run"
```

Comment:
How about configuring them in Makefile directly? And `CARGO_` prefix is a bad choice as they are supposed to be defined by cargo.

Reply:
Renamed. The env is to keep it symmetric. In the document above we encourage user to use `scripts/test`, which can't be set in makefile.